### DISCUSSION TITLE
Change the router used

### DISCUSSION THREAD

**Discussion by p-pichet** [GoogleCloudPlatform/buildpacks#448] [Q&A]
Hello. 

I recently found that the router used by the Google Cloud Function has been change. 

You can see [here](https://github.com/GoogleCloudPlatform/buildpacks/commit/342b274313c56b2f7e25c936714075c1923a3726)   that we don't use the bin/router but the router in the cloud-functions-framework.

I'm using the[ php-runtime/google-cloud ](https://github.com/php-runtime/google-cloud)  in a symfony project. We used the bin/router that call the phpruntime. 

Why do you change ?

**Comment by p-pichet** [comment]
@kennethye1 ?

**Comment by kennethye1** [comment]
Hi its because we upgraded composer to 2.2. 

You can see 
https://github.com/composer/composer/issues/10533
https://github.com/GoogleCloudPlatform/functions-framework-php/issues/130#issuecomment-1735940485

**Reply by kennethye1** [reply]
If you think this is a bug, please file a issue with context and how to reproduce this issue.
